website/routes.py

```diff
@@ -251,14 +250,13 @@ def ember_app(path=None):
 
     return send_from_directory(ember_app_folder, fp)
 
-@ember_flag_is_active('ember_home_page')
 def goodbye():
     # Redirect to dashboard if logged in
+    redirect_url = util.web_url_for('index')
     if _get_current_user():
-        return redirect(util.web_url_for('index'))
-    status.push_status_message(language.LOGOUT, kind='success', trust=False)
-    return {}
-
+        return redirect(redirect_url)
```

Comment:
This should still work. If you navigate directly to the goodbye route, while logged in, you should be sent back to your dash.